src/connectors/triplej.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+Connector.playerSelector = '#main-content';
+
+Connector.getArtist = () => {
+	return $('.playingNow div[class*="songArtist"]').text() || $('.view-liveMusicNow .artist').text();
```

Comment:
This can be replaced by an array of selectors.

Reply:
Got an example where we do similar?

Reply:
Figured